package/pbt.toml

```
@@ -8,7 +8,7 @@ Python Build Tool: A BusyBox that provides `python`, `pip`, `pex`, `pex3` and `p
 version = "0.7.0"
 
 [lift.scie-jump]
```

The issue is I wrote this badly from the get go. Should be `[lift.scie_jump]`. See the reference: https://science.scie.app/_/manifest/lift.html#scie-jump

---

Ah, thanks, that short cuts the investigation I hadn't got to! 🎉 

```suggestion
[lift.scie_jump]
```

---

To fix this sort of thing going forward: https://github.com/a-scie/lift/issues/46